index.html

```diff
@@ -685,6 +714,22 @@ <h3>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitud
 </pre>
             </section>
         </section>
+        <section>
+            <h2><code>DeviceProperties</code> interface</h2>
+            <pre class="idl">
+                [Exposed=PointerEvent]
+                interface DeviceProperties {
+                    constructor(long uniqueId);
+                    readonly attribute long uniqueId;
+                };
+            </pre>
+            <dl data-dfn-for="DeviceProperties" data-link-for="DeviceProperties">
+                <dt><dfn>uniqueId</dfn></dt>
+                    <dd>
+                        <p>A unique identifier for the pointing device. Pointer events generated from the same physical device will have the same <code>uniqueId</code> for the browsing session. A generic <code>uniqueId</code> value of <code>1</code> MUST be reserved for events generated by the primary mouse device. The <code>uniqueId</code> value of <code>-1</code> MUST be reserved and used to indicate events whose generating device could not be identified. <code>uniqueId</code> may be omitted/unset if hardware constraints prevent the UA from obtaining the device identifier. Like <a href="#dom-pointerevent-pointerid">pointerId</a>, to minimize the chance of fingerprinting and tracking across different pages or domains, the <code>uniqueId</code> MUST only be associated explicitly with that particular pointing device for the lifetime of the page / session, and a new randomized <code>uniqueId</code> MUST be chosen the next time that particular pointing device is used again in a new session.</p>
+                    </dd>
```

Comment:
It is a bit weird that value 1 is reserved for mouse, when pointerId for mouse can be 1 or 0. This inconsistency could easily lead to issues.

Reply:
I made this decision based on [this chain](https://groups.google.com/a/chromium.org/g/blink-dev/c/3eU-AHH8x4k/m/a0RpPWeUCAAJ) of feedback in a different forum. I understand both concerns, but it does seem more natural to me that only one value is reserved for mouse. What do you think? Happy to consider the alternative.

Reply:
It sounds like the initial feedback on that thread may have come from a position of not knowing about the 0 or 1 strangeness in the pointerevents spec. That said, given uniqueId is not really the same as pointerId I could see even not reserving a particular uniqueId for it.

Reply:
I understand that as pointerId has the same behavior, having this for uniqueId can be redundant. Are we all in agreement for not reserving a uniqueId for mouse? If so, I can remove that part of the text.

Reply:
Also, the text isn't too clearly explaining that the id isn't necessarily available on down event. Because that behavior is extremely surprising, there needs to be some warning about that in the spec.

Reply:
A proper test for this that isn't potentially flaky will be challenging since 0 or 1 are valid id choices even when not reserved. I suppose though if you have a few other events with uniqueIds first it will be unlikely that you get 0 when the first mouse event shows up.

Reply:
I have updated the text to remove the mouse specific id, as well as added a note with a warning on potential missing uniqueId. Please take another look :)

Reply:
For the web-platform-test, are you all comfortable with the idea of having some manual tests to start? The reason I ask is because testdriver does not yet support injecting a pointer event with a device id.

Reply:
To clarify, I have already added a couple of automated wpt tests for the invalid case and PointerEventInit testing as well.

Reply:
Closing this thread.